src/main/java/uk/gov/hmcts/ccd/domain/types/DocumentValidator.java

```diff
@@ -58,7 +55,7 @@ public BaseType getType() {
         }
 
         final String documentUrlValue = documentUrl.textValue();
-        final String urlPatternString = applicationParams.getValidDMDomain() + "/documents/[A-Za-z0-9-]+(?:/binary)?";
+        final String urlPatternString = "http.*/documents/[A-Za-z0-9-]+(?:/binary)?";
```

Comment:
See here for more details on document store's way of managing document urls in its responses:
https://tools.hmcts.net/confluence/pages/viewpage.action?spaceKey=RCCD&title=XFH+Support+for+CCD+Rest+Endpoints